README.md

```diff
@@ -5,4 +5,9 @@
 - `Stripped`: scan files that may be executable and report whether they are a Go executable that has had its symbols stripped.
 - `ImportHash`: calculate the [imphash](https://www.fireeye.com/blog/threat-research/2014/01/tracking-malware-import-hashing.html) of an executable with dynamic imports.
 - `GoSymbolHash`: calculate an imphash analogue for Go executables compiled by the gc-compiler.
-- `Sections`: provide section size statistics for an executable.
+
+    The `GoSymbolHash` algorithm is analogous to the algorithm described for `ImportHash` with the exception that Go's static symbols are used in place of the dynamic import symbols used by `ImportHash`.
+
+    The list of symbols referenced by the executable is obtained and the MD5 hash of the ordered list of symbols, separated by commas, is calculated.
```

Comment:
Is this in the order that they are listed in the binary?

Reply:
This is not specified in the original description of imphash (irritatingly), but is shown in the reference implementation. Yes, is is in appearance order in the binary without additional lexical ordering. Given that I find it irritating that it's not mentioned in imphash, I'll add that detail here.